docs/manuals/running_services/index.md

```diff
@@ -0,0 +1,90 @@
+# Running Taipy services
+
+Taipy provides you three services: Taipy Gui, Taipy Rest, and Taipy Core. You can run Taipy services by calling the method _run()_ from the service instance either `Gui^`, `Rest^`, or `Core^`. You can also use `taipy.run()^` to initiate the service.
+
+For example, you can run the Gui service with the following code:
+```python
+import taipy as tp
+
+if __name__ == "__main__":
+    gui = tp.Gui(page="# Getting started with *Taipy*")
+
+    # Approach 1
+    gui.run(gui, title="Taipy Demo")
+
+    # Approach 2
+    tp.run([gui], title="Taipy Demo")
+```
+
+By default, when running the Rest service, you will also start the Core service as the Rest service relies on the Core service to function:
+```python
+import taipy as tp
+
+if __name__ == "__main__":
+    rest = tp.Rest()
+
+    # Approach 1
+    rest.run()
+
+    # Approach 2
+    tp.run([rest])
+```
+
+The above code is the same as:
+```python
+import taipy as tp
+
+if __name__ == "__main__":
+    core = tp.Core()
+    rest = tp.Rest()
+
+    # Approach 1
+    core.run()
+    rest.run()
+
+    # Approach 2
+    tp.run([core, rest], title="Taipy Demo")
+```
+
+You can run the Core service individually with the below code:
+```python
+import taipy as tp
+
+if __name__ == "__main__":
+    core = tp.Core()
+
+    # Approach 1
+    core.run()
+
+    # Approach 2
+    tp.run([core])
```

Comment:
Is it possible to remove the mandatory array creation?

Reply:
@FabienLelaquais ah yes you can! I'll remove the array creation then